gsoc/2020/mentors.md

```diff
@@ -8,3 +8,6 @@ layout: plain
 * Ulrik Egede [ulrik.egede@monash.edu](mailto:ulrik.egede@monash.edu) Monash University
 * Alex Richards [a.richards@imperial.ac.uk](mailto:a.richards@imperial.ac.uk) Imperial College London
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) Imperial College London
+* Valentin Kuznetsov [vkuznet@gmail.com](mailto:vkuznet@gmail.com) Cornell University, USA
+* Federica Legger [federica.legger@to.infn.it](mailto:federica.legger@to.infn.it) Universita e INFN Torino
+* Christian Ariza [christian.ariza@gmail.com](mailto:christian.ariza@gmail.com) CERN
```

Comment:
the name of Christian Ariza should come first (sort by last name), and the new added names have to be sorted with respect to the existing ones in the same list.